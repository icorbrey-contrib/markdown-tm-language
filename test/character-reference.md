a &#123; b

c &#x12a3; d

e &amp; f

g &xxx; h.
