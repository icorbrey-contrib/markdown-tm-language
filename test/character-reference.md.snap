>a &#123; b
#^^ source.md meta.paragraph.md
#  ^ source.md meta.paragraph.md constant.character-reference.numeric.decimal.html punctuation.definition.character-reference.begin.html
#   ^ source.md meta.paragraph.md constant.character-reference.numeric.decimal.html punctuation.definition.character-reference.numeric.html
#    ^^^ source.md meta.paragraph.md constant.character-reference.numeric.decimal.html constant.numeric.integer.decimal.html
#       ^ source.md meta.paragraph.md constant.character-reference.numeric.decimal.html punctuation.definition.character-reference.end.html
#        ^^^ source.md meta.paragraph.md
>
>c &#x12a3; d
#^^ source.md meta.paragraph.md
#  ^ source.md meta.paragraph.md constant.character-reference.numeric.hexadecimal.html punctuation.definition.character-reference.begin.html
#   ^ source.md meta.paragraph.md constant.character-reference.numeric.hexadecimal.html punctuation.definition.character-reference.numeric.html
#    ^ source.md meta.paragraph.md constant.character-reference.numeric.hexadecimal.html punctuation.definition.character-reference.numeric.hexadecimal.html
#     ^^^^ source.md meta.paragraph.md constant.character-reference.numeric.hexadecimal.html constant.numeric.integer.hexadecimal.html
#         ^ source.md meta.paragraph.md constant.character-reference.numeric.hexadecimal.html punctuation.definition.character-reference.end.html
#          ^^^ source.md meta.paragraph.md
>
>e &amp; f
#^^ source.md meta.paragraph.md
#  ^ source.md meta.paragraph.md constant.character-reference.named.html punctuation.definition.character-reference.begin.html
#   ^^^ source.md meta.paragraph.md constant.character-reference.named.html keyword.control.character-reference.html
#      ^ source.md meta.paragraph.md constant.character-reference.named.html punctuation.definition.character-reference.end.html
#       ^^^ source.md meta.paragraph.md
>
>g &xxx; h.
#^^^^^^^^^^^ source.md meta.paragraph.md
>